.github/workflows/container_tests.yml

```diff
@@ -17,10 +17,10 @@ jobs:
         python: [3.7]
       fail-fast: false
     steps:
-    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # 4.2.2
+    - uses: actions/checkout@v4
```

Comment:
Using a specific commit is deliberate, out of abundance of care.

We'll only update to newer versions/commits when really needed, so please change back.